common/history/states/far_east_asian_states.txt

```diff
@@ -155,15 +155,15 @@ STATES = {
 			}
 		}
 		
-		add_claim = c:CHI add_homeland = chinese
+		add_claim = c:CHI add_homeland = chinese add_homeland = han
```

Comment:
han is to be consolidated into chinese for game performance reasons so there should be no additional homeland.